modules/src/ics23_commitment/mod.rs

```diff
@@ -32,6 +32,10 @@ impl CommitmentProof {
 */
 
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
-pub struct CommitmentPrefix;
+pub struct CommitmentPrefix(Vec<u8>);
```

Comment:
Wondering if we should have a formatter for this. The output of the query shows: `prefix: CommitmentPrefix([112, 114, 101, 102, 105, 120]) }` (decimal), in the genesis and curl it's `cHJlZml4` (base64), ascii is `"prefix"`

Reply:
Implemented a fmt::Debug trait to properly format the CommitmentPrefix output in this [commit](https://github.com/informalsystems/ibc-rs/pull/136/commits/f8c0721250f770bc5bdd67b8cdd6ea92532369c6)